.github/ISSUE_TEMPLATE/1.bug-report.yml

```
@@ -1,39 +1,50 @@
-name: Report an issue
-description: Report an issue with Zed.
+name: Report a bug
+description: Report a problem with Zed.
 type: Bug
 labels: "state:needs triage"
 body:
   - type: markdown
     attributes:
       value: |
-        Feature requests should be opened in [discussions](https://github.com/zed-industries/zed/discussions/new/choose).
+        Is this bug already reported? Upvote to get it noticed faster. [Here's the search](https://github.com/zed-industries/zed/issues). Upvote means giving it a :+1: reaction.
+        
+        Feature request? Please open in [discussions](https://github.com/zed-industries/zed/discussions/new/choose) instead.
 
-        Before opening a new issue, please do a [search](https://github.com/zed-industries/zed/issues) of existing issues and :+1: upvote the existing issue instead. This will help us maintain a proper signal-to-noise ratio.
-
-        If you need help with your own project, you can ask a question in our [Discord Support Forums](https://discord.com/invite/zedindustries).
+        Just have a question or need support? Welcome to [Discord Support Forums](https://discord.com/invite/zedindustries).
   - type: textarea
     attributes:
       label: Reproduction steps
-      description: A step-by-step description of how to reproduce the issue from a **clean Zed install**. Any code must be sufficient to reproduce (make sure to include context!). Include code as text, not just as a screenshot. **Issues with insufficient detail may be summarily closed**.
+      description: A step-by-step description of how to reproduce the bug from a **clean Zed install**. The more context you provide, the easier it is to find and fix the problem fast.
       placeholder: |
         1. Start Zed
         2. Click X
-        3. Y will happen
     validations:
       required: true
   - type: textarea
     attributes:
       label: Current vs. Expected behavior
       description: |
-        A clear and concise description of what is the current behavior (screenshots, videos), vs. what you expected the behavior to be.
+        Current behavior (screenshots, videos, etc are appreciated), vs. what you expected the behavior to be.
```

```suggestion
        Current behavior (e.g., screenshots, videos, are appreciated), vs. what you expected the behavior to be.
```

Pedantic nitpick time but I think `e.g.` fits better here, since `etc.` (which always required a `.`) [should terminate a clause](https://www.grammarly.com/blog/commonly-confused-words/et-cetera-etc/).